stn/node.py

```diff
@@ -1,9 +1,10 @@
+from stn.utils.uuid import generate_uuid
 
 
 class Node(object):
     """Represents a timepoint in the STN """
 
-    def __init__(self, task_id='', pose='', type='zero_timepoint'):
+    def __init__(self, task_id=generate_uuid(), pose='', type='zero_timepoint'):
```

Comment:
should this be an optional argument? I think this is always linked to a task, right?